dev/package-examples/yamlpipeline-1.0.0/dataset/log/elasticsearch/ingest-pipeline/pipeline-entry.yml

```diff
@@ -0,0 +1,113 @@
+---
+description: Pipeline for normalizing Kubernetes CoreDNS logs.
+processors:
+  - pipeline:
+      if: ctx.message.charAt(0) == (char)("{")
+      name: '{{IngestPipeline "pipeline-json" }}'
+  - pipeline:
+      if: ctx.message.charAt(0) != (char)("{")
+      name: '{{IngestPipeline "pipeline-plaintext" }}'
```

Comment:
@skh We should still decide if we really want to use `{{IngestPipeline` as our identifier or should use something differently. In Beats we used `{<`.

Reply:
Then we should probably use `{<` too, because then the pipelines can be used from both projects. We could even do _both_.